contracts/oracle/OracleRouter.sol

```diff
@@ -57,13 +57,17 @@ contract OracleRouter is OracleRouterBase {
             return address(0x51D7180edA2260cc4F6e4EebB82FEF5c3c2B8300);
         } else if (
             // USDCe
-            asset == address(0xA7D7079b0FEaD91F3e65f86E8915Cb59c1a4C664)
+            asset == address(0xA7D7079b0FEaD91F3e65f86E8915Cb59c1a4C664) ||
```

Comment:
Open to discussion of the right design choice here as USDC and USCDe would be equivalent (even though they are not )